### PR TITLE
feature/AOS-2925

### PR DESCRIPTION
**Pull request by bjartek**
- sleep 1 second between affiliations are crawled

Boyscout rule
- parameterize crawler
- update dependencies
- remove invalid doc

### CODE REVIEW

File: src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt [hunk 1]
```diff
@@ -31,29 +31,22 @@ class ApplicationDataServiceOpenShift(
     val addressService: AddressService,
     val imageService: ImageService
 ) : ApplicationDataService {
-    override fun findAllVisibleAffiliations(): List<String> {
-        throw NotImplementedError("findAllVisibleAffiliations is not supported")
-    }
 
     val mtContext = newFixedThreadPoolContext(6, "mokeyPool")
```
Comment (jarlehansen) [inline src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt]: I am getting a warning for this and when I look at the docs its says:
This API will be replaced in the future.

File: src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt [hunk 2]
```diff
@@ -31,29 +31,22 @@ class ApplicationDataServiceOpenShift(
...
-            val allEnvironments = findAllEnvironments()
-            val environmentsForAffiliations = allEnvironments.filter { affiliations.contains(it.affiliation) }
-            findAllApplicationDataByEnvironments(environmentsForAffiliations)
-        }
-        return apps.filter { if (ids.isEmpty()) true else ids.contains(it.applicationDeploymentId) }
+        val affiliationGroups = findAndGroupAffiliations(affiliations)
+        return findAllApplicationDataForEnv(ids, affiliationGroups)
+    }
+
+    override fun findAllVisibleAffiliations(): List<String> {
```
Comment (jarlehansen) [inline src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt]: This is a general comment about the design of the decorator, and not specifically the changes in the PR.
I feel we are moving in a direction here that I do not like. The implementations of `ApplicationDataService` are becoming more and more different, even providing separate functions. For me this should be split into two separate classes at this point, for me the way it is now is just confusing to have this as a decorator.

File: src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt (hunk 1 above)
Comment (bjartek) [reply src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt]: Yes, but there is no alternative now AFAIK so we just leave it in there.

File: src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt (hunk 2 above)
Comment (bjartek) [reply src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt]: I think we should discuss this at a Team meeting.

**Pull request merged**